bot_cli/commands.py

```diff
@@ -227,13 +226,17 @@ def show_all_contacts():
         result = list()
         for record in address_book.data.values():
             result.append(str(record))
-        print(Fore.LIGHTBLUE_EX + "\n".join(result))
+        print_info("\n".join(result))
     else:
-        print(Fore.LIGHTBLUE_EX + "No contacts have been added yet")
+        print_info("No contacts have been added yet")
 
 
 @max_period_error
 def birthdays(args):
+    """
+    Show all birthdays in defined period
```

Comment:
...upcoming birthdays...
default period is 7 days

Reply:
done